### PR TITLE
fix and update k8s deploy

### PR DESCRIPTION
**Pull request by rockxsj**
## *Tips*
- *Thanks very much for contributing to Apache DolphinScheduler.*
- *Please review https://dolphinscheduler.apache.org/en-us/community/index.html before opening a pull request.*

## What is the purpose of the pull request

    update Dockerfile to add mysql-client for mysql database service test
    update wechat alert config replace placeholder from to {xx} because of the will be treat as env variable
    fix unit test of EnterpriseWeChatUtilsTest
    update values.yaml default config for helm deploy DS cluster
    update values.yaml default config for custom install
    update templates of helm for custom install
    update some name of env varibale, for example change from POSTGRESQL_HOST to DATABASE_HOST
    delele the redundancy requirements.yaml for heml
    update README* in the docker dir and subdir
    update startup scripts for support mysql as backend database

## Brief change log

Modify ambari_plugin/common-services/DOLPHIN/1.2.1/package/scripts/params.py
Modify ambari_plugin/common-services/DOLPHIN/1.3.0/package/scripts/params.py
Modify docker/build/Dockerfile
Modify docker/build/README.md
Modify docker/build/README_zh_CN.md
Modify docker/build/conf/dolphinscheduler/alert.properties.tpl
Modify docker/build/conf/dolphinscheduler/common.properties.tpl
Modify docker/build/conf/dolphinscheduler/datasource.properties.tpl
Modify docker/build/conf/dolphinscheduler/logback/logback-alert.xml
Modify docker/build/conf/dolphinscheduler/zookeeper.properties.tpl
Modify docker/build/startup-init-conf.sh
Modify docker/build/startup.sh
Modify docker/docker-swarm/docker-compose.yml
Modify docker/docker-swarm/docker-stack.yml
Modify docker/kubernetes/dolphinscheduler/Chart.yaml
Modify docker/kubernetes/dolphinscheduler/README.md
Modify docker/kubernetes/dolphinscheduler/requirements.yaml
Modify docker/kubernetes/dolphinscheduler/templates/NOTES.txt
Modify docker/kubernetes/dolphinscheduler/templates/_helpers.tpl
Modify docker/kubernetes/dolphinscheduler/templates/configmap-dolphinscheduler-alert.yaml
Add docker/kubernetes/dolphinscheduler/templates/configmap-dolphinscheduler-common.yaml
Modify docker/kubernetes/dolphinscheduler/templates/deployment-dolphinscheduler-alert.yaml
Modify docker/kubernetes/dolphinscheduler/templates/deployment-dolphinscheduler-api.yaml
Modify docker/kubernetes/dolphinscheduler/templates/deployment-dolphinscheduler-frontend.yaml
Modify docker/kubernetes/dolphinscheduler/templates/statefulset-dolphinscheduler-master.yaml
Modify docker/kubernetes/dolphinscheduler/templates/statefulset-dolphinscheduler-worker.yaml
Modify docker/kubernetes/dolphinscheduler/values.yaml
Modify dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/utils/EnterpriseWeChatUtils.java
Modify dolphinscheduler-alert/src/main/resources/alert.properties
Modify dolphinscheduler-alert/src/test/java/org/apache/dolphinscheduler/alert/utils/EnterpriseWeChatUtilsTest.java
Modify dolphinscheduler-ui/package.json
Modify script/dolphinscheduler-daemon.sh
Modify docker/docker-swarm/docker-compose.yml
Modify docker/docker-swarm/docker-stack.yml

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

File: docker/build/Dockerfile [hunk 1]
```diff
@@ -42,8 +42,8 @@ ADD ./apache-dolphinscheduler-incubating-${VERSION}-dolphinscheduler-bin.tar.gz
 RUN mv /opt/apache-dolphinscheduler-incubating-${VERSION}-dolphinscheduler-bin/ /opt/dolphinscheduler/
 ENV DOLPHINSCHEDULER_HOME /opt/dolphinscheduler
 
-#4. install pg
-RUN apk add postgresql postgresql-contrib
+#4. install database
+RUN apk add postgresql postgresql-contrib mysql-client
```
Comment (khadgarmage) [inline docker/build/Dockerfile:46]: mysql's license can not meet the requirements of apache, pls remove it

File: docker/build/startup.sh [hunk 2]
```diff
@@ -22,24 +22,32 @@ DOLPHINSCHEDULER_BIN=${DOLPHINSCHEDULER_HOME}/bin
...
         sleep 5
     done
 
-    echo "connect postgresql service"
-    v=$(sudo -u postgres PGPASSWORD=${POSTGRESQL_PASSWORD} psql -h ${POSTGRESQL_HOST} -p ${POSTGRESQL_PORT} -U ${POSTGRESQL_USERNAME} -d dolphinscheduler -tAc "select 1")
-    if [ "$(echo '${v}' | grep 'FATAL' | wc -l)" -eq 1 ]; then
-        echo "Error: Can't connect to database...${v}"
-        exit 1
+    echo "connect ${DATABASE_TYPE} service"
+    if [ ${DATABASE_TYPE} = "mysql" ]; then
```
Comment (khadgarmage) [inline docker/build/startup.sh:39]: license problem

File: docker/build/Dockerfile (hunk 1 above)
Comment (liwenhe1993) [reply docker/build/Dockerfile:46]: +1

**Comment by dailidong** [comment]
@liwenhe1993 @khadgarmage , I see rockxsj has updated as you said , I will merge this PR

**Pull request merged**